lib/wasi/src/syscalls/wasix/call_close.rs

```diff
@@ -0,0 +1,10 @@
+use super::*;
+use crate::syscalls::*;
+
+// FIXME: remove , since it's no longer used.
```

Comment:
So this need to be removed? Shall I do it?

Reply:
Removed